Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -2730,10 +2730,17 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
       case .blockComment(let text):
         if index > 0 || isStartOfFile {
           appendToken(.comment(Comment(kind: .block, text: text), wasEndOfLine: false))
-          // We place a size-0 break after the comment to allow a discretionary newline after the
-          // comment if the user places one here but the comment is otherwise adjacent to a text
-          // token.
-          appendToken(.break(.same, size: 0))
+          // There is always a break after the comment to allow a discretionary newline after it.
+          // This break's size is the number of spaces after the comment to maintain original spaces
```

Comment:
Is there a reason to preserve the original spacing instead of forcing it to be one?

Reply:
I'm not aware of any conventions where you have multiple spaces between the comment and the text token, unless the author was trying to use spaces for alignment. I think we discourage manual alignment so it's probably not important to support that. I updated it to only allow 1 space.